php/class-post-type.php

```diff
@@ -322,6 +332,8 @@ public function render_data_metabox( $post ) {
 		echo sprintf( '%1$s %2$s %3$s', esc_html__( 'Modified:', 'customize-snapshots' ), esc_html( get_the_modified_date( '' ) ), esc_html( get_the_modified_time( '' ) ) ) . '<br>';
 		echo '</p>';
 
+		$fork_markup = '<a href="#" id="snapshot-fork" class="button button-secondary" data-post-id="' . $post->ID . '" data-nonce="' . wp_create_nonce( 'snapshot-fork' ) . '">' . esc_html__( 'Fork', 'customize-snapshots' ) . '</a>';
```

Comment:
This should be a button not a link and also using `sprintf` would clean things up. Something like:

```
$fork_markup = sprintf(
	'<button id="snapshot-fork" class="button button-secondary" data-post-id="%s" data-nonce="%s">%s</button><span class="spinner snapshot-fork-spinner"></span>',
	$post->ID,
	wp_create_nonce( 'snapshot-fork' ),
	esc_html__( 'Fork', 'customize-snapshots' )
);
```